openqabot/loader/qem.py

```diff
@@ -115,21 +115,19 @@ def get_incident_settings_data(token: Dict[str, str], number: int) -> Sequence[D
     if "error" in data:
         raise ValueError
 
-    ret = []
-    for d in data:
-        ret.append(
-            Data(
-                number,
-                d["id"],
-                d["flavor"],
-                d["arch"],
-                d["settings"]["DISTRI"],
-                d["version"],
-                d["settings"]["BUILD"],
-                "",
-            )
+    ret = [
```

Comment:
Maybe also remove the variable completely and return the rvalue directly?

Reply:
Sure, done